teps/0114-custom-tasks-beta.md

```diff
@@ -208,6 +208,10 @@ Remove guarding of `Custom Tasks` behind `enable-custom-tasks` and `enable-api-f
 When [TEP-0096: Pipelines V1 API][tep-0096] is implemented to add V1 API, `Custom Tasks` will be gated behind feature
 gate `enable-api-fields` being set to `"beta"` - this is out of scope for this TEP  (in scope for TEP-0096).
 
+##### Cancellation
+
+Implementing cancellation is now `required` instead of `optional`. If Custom Task Implementor does not implement `Cancellation`, pipeline-level timeout and PipelineRun cancellation will not work properly.
```

Comment:
A bit of a wording suggestion: 

```suggestion
Implementing cancellation is now `required` instead of `optional`. If the Custom Task implementor does not support cancellation via `.spec.status`, `Pipeline` timeouts and `PipelineRun` cancellation will not work properly.
```

Reply:
Thanks @abayer. 

I reviewed TEP-0002 and it didn't explicitly tell users that how do we set the `spec.Status` and `spec.StatusMessage` and how do we expect custom task implementors signal back. I'll need to update this TEP to make that explicit.